beacon-chain/sync/service.go

```diff
@@ -87,13 +87,6 @@ func (r *RegularSync) Status() error {
 	return nil
 }
 
-// ClearPendingBlocks clears outstanding pending blocks waiting to be processed,
```

Comment:
Ah yes. This was there so we could clear the pending blocks cache when there's a finalized epoch so we can "pivot" to the next finalized check point. 

It's ok to delete the unused code now because we dont have the pivot feature yet